src/ssl/support.cc

```diff
@@ -1367,9 +1367,9 @@ void Ssl::InRamCertificateDbKey(const Ssl::CertificateProperties &certProperties
 {
     bool origSignatureAsKey = false;
     if (certProperties.mimicCert.get()) {
-        ASN1_BIT_STRING *sig = nullptr;
+        const_ASN1_BIT_STRING *sig = nullptr;
 #if HAVE_LIBCRYPTO_X509_GET0_SIGNATURE
-        X509_ALGOR *sig_alg;
+        const_X509_ALGOR *sig_alg;
```

Comment:
Please use this opportunity to initialize `sig_alg` (to nullptr) before passing it to an external function.

Reply:
done